internal/common/disasters/environmentdefinition.go

```diff
@@ -32,6 +32,7 @@ type disasterParameters struct {
 type DisasterReport struct {
 	Magnitude shared.Magnitude
 	X, Y      shared.Coordinate
+	Effects   DisasterEffects
 }
```

Comment:
Wondering if this is copied correctly as part of GameState?

Reply:
structs are reference types

Reply:
It is in the gamestate but it seems that the first entry is null for the Effects fields...

Reply:
The copy works correctly, but I the DisasterReport does not have a default value so the first round copies nulls.